api/capi/src/nnstreamer-capi-single.c

```diff
@@ -954,3 +954,56 @@ ml_single_invoke_dynamic (ml_single_h single,
 
   return status;
 }
+
+/**
+ * @brief Sets the property value for the given model.
+ */
+int
+ml_single_set_property (ml_single_h single, const char *name, const char *value)
+{
+  ml_single *single_h;
+  GObject *filter_obj;
+
+  check_feature_state ();
+
+  if (!single || !name || !value)
+    return ML_ERROR_INVALID_PARAMETER;
+
+  ML_SINGLE_GET_VALID_HANDLE_LOCKED (single_h, single, 0);
+
+  filter_obj = G_OBJECT (single_h->filter);
+
+  /** @todo fill here (allow all properties?) */
+  g_object_set (filter_obj, name, value, NULL);
```

Comment:
As not all properties of tensor_filter are string (some are boolean), do we allow setting those properties here by parsing them manually or setting them is not allowed?

Reply:
IMO, it is better to allow only the properties of tensor info (input,output, inputname, inputtype, ... ) and accelerator.
We cannot change model path and framework after opening the single-shot instance.

Reply:
Right.
However what about properties like `is-updatable`. It expects a boolean value. 
Such properties will also be supported? (will require parsing input `char *` to `boolean`).

Reply:
Allowed property name:
input, inputtype, inputname, inputlayout
output, outputtype, outputname, outputlayout
accelerator
is-updatable

For the case of boolean type: parse string value (true/false)